axelor-redmine/src/main/java/com/axelor/apps/redmine/service/common/RedmineCommonService.java

```diff
@@ -219,8 +213,8 @@ protected void updateTransaction() {
 
     JPA.clear();
 
-    if (!JPA.em().contains(batch)) {
-      batch = JPA.find(Batch.class, batch.getId());
+    if (!JPA.em().contains(methodParameters.getBatch())) {
+      methodParameters.setBatch(JPA.find(Batch.class, methodParameters.getBatch().getId()));
```

Comment:
Store `methodParameters.getBatch()` inside a variable in order to avoid code duplication
Please, do it everywhere a same getter is called multiple times